seedvault/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,26 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <queries>
+        <!-- Most Seed Vault Wallet ACTION_*'s use explicit intents targeted to this package -->
         <package android:name="com.solanamobile.seedvaultimpl" />
+
```

Comment:
Add all the various contract-defined interfaces to the `<queries>` list. There is no guarantee that all of these intents and content providers are implemented by the canonical `com.solanamobile.seedvaultimpl` package.